exercises/bhuvan_bezawada/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,151 @@
+/**
+ * PROGRAM DESCRIPTION:
+ *
+ * A command line program to read in a text file and give the occurrences of the
+ * words.
+ *
+ **/
+
+// Imports
+#include <algorithm>
+#include <fstream>
+#include <iostream>
+#include <iterator>
+#include <map>
+#include <string>
+#include <vector>
+
+// Namespace being used
+using namespace std;
+
+/**
+ * Code to flip the <string, int> pairs.
+ * Code adapted from:
+ *https://stackoverflow.com/questions/5056645/sorting-stdmap-using-value?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa
+ **/
+template <typename A, typename B> pair<B, A> flipPair(const pair<A, B> &p) {
+  return pair<B, A>(p.second, p.first);
+}
+
+template <typename A, typename B> multimap<B, A> flipMap(const map<A, B> &src) {
+  multimap<B, A> dst;
+  transform(src.begin(), src.end(), inserter(dst, dst.begin()), flipPair<A, B>);
+  return dst;
+}
+
+/**
+ *  Split function to take a line of input and return words in line.
+ *  From Accelerated C++ Textbook, page 88.
+ **/
+vector<string> split(const string &s) {
```

Comment:
This is acceptable for the purpose of the exercises, but you should be aware of `regular expressions` if you are not already as finding words would be much easier using them.